### PR TITLE
Ot217 27

### PR DESCRIPTION
**Pull request by filburt88**
* Reactive Form Added
* Dynamic title added
* Form responds according to modification/creation of News, using actual data as placeholder or leaving blank spaces
* Image convertion to Base64
* API communication working

Screens:
Mobile View
Desktop View
Form Completed

### CODE REVIEW

File: src/app/features/pages/news/news-form/news-form.component.ts [hunk 1]
```diff
@@ -1,15 +1,96 @@
...
+      category_id: this.news.get(["category_id"]).value,
+      content: this.news.get(["content"]).value,
+      image: this.cardImageBase64,
+    };
+
+    if (this.newsId) {
+      this.newsService
+        .updateNews(newsCommit, this.newsId)
+        .subscribe((response) => {
+          console.log(response);
```
Comment (anagon88) [inline src/app/features/pages/news/news-form/news-form.component.ts:46]: borrar console log

File: src/app/features/pages/news/news-form/news-form.component.ts [hunk 2]
```diff
@@ -1,15 +1,96 @@
...
+  ngOnInit(): void {
+    this.categoriesService.getCategories().subscribe((response) => {
+      this.categories = response.data;
+    });
+
+    if (this.newsId) {
+      this.title = "Modificar Novedad";
+      this.newsService.getNewsId(this.newsId).subscribe((response) => {
+        this.currentNews.push(response.data);
+        console.log(this.currentNews[0]);
```
Comment (anagon88) [inline src/app/features/pages/news/news-form/news-form.component.ts:75]: borrar todos los console logs

**Review by anagon88 — CHANGES_REQUESTED** [review]

File: src/app/features/pages/news/news-form/news-form.component.ts (hunk 1 above)
Comment (filburt88) [reply src/app/features/pages/news/news-form/news-form.component.ts]: Borrados.

File: src/app/features/pages/news/news-form/news-form.component.ts (hunk 2 above)
Comment (filburt88) [reply src/app/features/pages/news/news-form/news-form.component.ts]: Borrados

**Pull request merged**